retis/Cargo.toml

```diff
@@ -13,7 +13,7 @@ include = ["src/", "build.rs", "profiles/",  "LICENSE"]
 edition = "2021"
 
 [package.metadata.misc]
-release_name = "senyoret"
+release_name = "panettone"
```

Comment:
The series LGTM, @amorenoz comment was addressed.
Approved and merged. Enjoy! :)